dm_control/utils/inverse_kinematics.py

```diff
@@ -35,7 +35,7 @@
 
 
 def qpos_from_site_pose(physics,
-                        site_name,
+                        sites_names,
```

Comment:
This is a widely used function, that we don't want to break API compatibility on.

Can you instead add a new function, qpos_from_site_poses which takes a parameter (called site_names rather than sites_names, BTW)?

The existing `qpos_from_site_pose` function could call down to the new `qpos_from_site_poses`.